packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -81,6 +81,25 @@ export class MirrorNodeClient {
     private static CONTRACT_RESULT_LOGS_PROPERTY = 'logs';
     private static CONTRACT_STATE_PROPERTY = 'state';
 
+    static acceptedErrorStatusesResponsePerRequestPathMap: Map<string, Array<number>> = new Map([
+        [MirrorNodeClient.GET_ACCOUNTS_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BALANCE_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BLOCK_ENDPOINT, [400, 404]],
+        [MirrorNodeClient.GET_BLOCKS_ENDPOINT, [400, 404]],
```

Comment:
We will never return a 404 for any list endpoint.

Reply:
what you mean by `list` endpoint?

Reply:
Many of the API endpoints return lists vs a single item e.g. `api/v1/accounts` return a list but `api/v1/accounts/{accountId}` will return a single object.
The former would return an empty list on no data but the latter would return a 404.
We should only look 404's on the non list endpoints.

Granted some business logic down the line may hint to a retry if a list endpoint returned nothin when we expect some data e.g. calling for transactions before mirror node processed the block for an accounts 1st transaction